src/AudioManager.ts

```diff
@@ -0,0 +1,124 @@
+import MediaAudio from "./MediaAudio";
+import AudioInputsService from "./AudioInputsService";
+import AudioOutputsService, {
+  AudioOutputsServicePlayOptions,
+} from "./AudioOutputsService";
+import BrowserSttService from "./BrowserSttService";
+
+export interface AudioManagerOptions {
+  duckVolumeLevel: number;
+  normalVolumeLevel: number;
+  sttService: "browser" | "deepgram";
+  streamTimeslice: number;
+}
+
+class AudioManager {
+  public audioInputsService: AudioInputsService;
```

Comment:
Shouldn't these be private? I want AudioManager to hide away anything that is not needed
audioInputsService
audioInputsBrowser
audioOutputsService
mediaAudio

In tests you can test the mocks rather than access though the Manager